src/tests/roc_pipeline/test_helpers/packet_reader.h

```diff
@@ -165,9 +167,9 @@ class PacketReader : public core::NonCopyable<> {
                           core::nanoseconds_t base_ts) {
         CHECK(pkt.rtp());
 
-        if (base_ts < 0) {
+        if (base_ts == 0) {
             LONGS_EQUAL(0, pkt.rtp()->capture_timestamp);
-        } else {
+        } else if (base_ts > 0) {
```

Comment:
This rules became quite a bit complicated.

Suggestion: revert changes in PacketReader and just don't use it in TEST(sender_sink, timestamp_mapping_remixing_packet_reader), instead read packets directly from queue and check their CTS.